test-requirements.txt

```diff
@@ -0,0 +1,4 @@
+mock
+odoo_test_helper
+odoo-addon-product-attribute-set @ git+https://github.com/OCA/odoo-pim.git@refs/pull/142/head#subdirectory=setup/product_attribute_set
+odoo-addon-attribute-set @ git+https://github.com/OCA/odoo-pim.git@refs/pull/145/head#subdirectory=setup/attribute_set
```

Comment:
Should not be merged. Only needed for the pipeline to pass while these PR aren't merged.